CONTRIBUTING.rst

```diff
@@ -0,0 +1,177 @@
+.. _developers-contributing:
+
+Contribute to WarpX
+===================
+
+We welcome new contributors!
+Here is how to participate to the WarpX development.
+
+Git workflow
+------------
+
+The WarpX project uses `git <https://git-scm.com>`_ for version control.
+If you are new to git, you can follow one of these tutorials:
+
+- `Learn git with bitbucket <https://www.atlassian.com/git/tutorials/learn-git-with-bitbucket-cloud>`_
+- `git - the simple guide <http://rogerdudler.github.io/git-guide/>`_
+
+Make your own fork and create a branch on it
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The basic WarpX workflow is:
+
+- Fork the main repo (or update it if you already created it).
+- Implement your changes and push them on a new branch ``<branch_name>`` on your fork.
+- Create a Pull Request from branch ``<branch_name>`` on your fork to branch `master` on the main WarpX repo.
+
+First, let us setup your local git repo. Make your own fork of the main (``upstream``) WarpX repo:
```

Comment:
Maybe it is clearer if the list between lines 23 and 25 is numbered (or a sub-sections list) and then these steps fall into each of those numbers. I can be wrong about this, but I think this way it is not as much jumping forth and back as much.

Reply:
Sounds good, I'll add 1.-3. for now :)